### PR TITLE
Update en.yml

### LINKED ISSUE #11424: Improve text for welcome contribution notice
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [config/locales/en.yml](https://github.com/publiclab/plots2/blob/main/config%2Flocales%2Fen.yml) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -1133,10 +1133,17 @@ en:
     logged_out: "Successfully logged out."
   notes_controller:
     research_notes: "Research notes"
-    thank_you_for_contribution: "Success! Thank you for contributing open research,
-      and thanks for your patience while your post is approved by <a href='/wiki/moderation'>community
-      moderators</a> and we'll email you when it is published. In the meantime, if
-      you have more to contribute, feel free to do so."
+    thank_you_for_contribution: |
+      <h2>Success! You did it! Hooray!</h2>
+      <hr>
+      <p>Thank you so much for contributing open research and being part of our community.</p>
+      <h2>Here's what's happening now:</h2>
+      <hr>
+      <p>Your <a href='/wiki/first-contribution'>first contribution</a> is awaiting approval by <a href='/wiki/moderation'>community moderators</a>. 
+      <b>If you have asked an urgent question about an environmental concern, we will prioritize reviewing it</b>.
+      Thank you for your patience, in the next hours your contribution should be public. We'll email you when it is published.
+      In the meantime, if you have more to contribute, feel free to do so. Visit the <a href='/wiki/welcome'>Welcome Page</a> to discover more ways to connect.
+      </p>
     thank_you_for_question: "Success! Thank you for contributing with a question, and
       thanks for your patience while your question is approved by <a href='/wiki/moderation'>community
       moderators</a> and we'll email you when it is published."
```

- [ ] 📝 **Update** the file [test/functional/notes_controller_test.rb](https://github.com/publiclab/plots2/blob/main/test/functional/notes_controller_test.rb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -296,7 +296,16 @@ class NotesControllerTest < ActionController::TestCase
          }
     # , :main_image => "/images/testimage.jpg"

-    assert_equal "Success! Thank you for contributing open research, and thanks for your patience while your post is approved by <a href='/wiki/moderation'>community moderators</a> and we'll email you when it is published. In the meantime, if you have more to contribute, feel free to do so.", flash[:notice]
+    assert_equal "<h2>Success! You did it! Hooray!</h2>
+      <hr>
+      <p>Thank you so much for contributing open research and being part of our community.</p>
+      <h2>Here's what's happening now:</h2>
+      <hr>
+      <p>Your <a href='/wiki/first-contribution'>first contribution</a> is awaiting approval by <a href='/wiki/moderation'>community moderators</a>.
+      <b>If you have asked an urgent question about an environmental concern, we will prioritize reviewing it</b>.
+      Thank you for your patience, in the next hours your contribution should be public. We'll email you when it is published.
+      In the meantime, if you have more to contribute, feel free to do so. Visit the <a href='/wiki/welcome'>Welcome Page</a> to discover more ways to connect.
+      </p>", flash[:notice]
     assert_nil flash[:warning] # no double notice
     assert_equal 4, Node.last.status
     assert_equal title, Node.last.title
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:
1. If you are not familiar with GitHub or the pull request model, [here is a guide you can follow](https://guides.github.com/activities/hello-world/) on how GitHub works.
2. If you are familiar with the terminal or would like to learn to use it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

3. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

### PR DESCRIPTION
**Pull request by priyda**
Changes for issue : Improve text for welcome contribution notice #11424"

Added required changes for improving text for welcome changes.
Fixes #11424

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1133,10 +1133,17 @@ en:
...
   notes_controller:
     research_notes: "Research notes"
-    thank_you_for_contribution: "Success! Thank you for contributing open research,
-      and thanks for your patience while your post is approved by <a href='/wiki/moderation'>community
-      moderators</a> and we'll email you when it is published. In the meantime, if
-      you have more to contribute, feel free to do so."
+    thank_you_for_contribution: |
+      <h2>Success! You did it! Hooray!</h2>
+      <hr>
+      <p>Thank you so much for contributing open research and being part of our community.</p>
```
Comment (TildaDares) [inline config/locales/en.yml:1139]:
```suggestion
      <p>Thank you so much for contributing to open research and being a part of our community.</p>
```

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @priyda, feel free to just click the `Commit suggestion` button. Thanks!

**Comment by TildaDares** [comment]
Hi @priyda, I also made changes to the issue. Do you think you could implement those changes? Sorry for the inconvenience!

**Comment by vamshidhar-kasulabada** [comment]
Hi @TildaDares Can I Work on this issue.

**Comment by vamshidhar-kasulabada** [comment]
HI @TildaDares Can I work on this issue #11424

**Comment by TildaDares** [comment]
Hi @vamshidhar-kasulabada, this is a first-timers only issue and someone is already working on it. Thank you!

File: config/locales/en.yml (hunk 1 above)
Comment (TildaDares) [reply config/locales/en.yml:1139]: Hi @priyda, do you need any help with this? We're here to help!

Comment (priyda) [reply config/locales/en.yml]: Hi, I have commited suggestion. Let me know if I need to make any more changes.

**Comment by priyda** [comment]
>

**Pull request closed without merging**